SPECS-EXTENDED/mock/mock.spec

```diff
@@ -0,0 +1,661 @@
+Summary: LVM plugin for mock
+Requires: %{name} = %{version}-%{release}
+Requires: lvm2
+
+%description lvm
+Mock plugin that enables using LVM as a backend and support creating snapshots
+of the buildroot.
+
+%package filesystem
+Summary:  Mock filesystem layout
+Requires(pre):  shadow-utils
+
+%description filesystem
+Filesystem layout and group for Mock.
+
+%prep
+%setup -q
+for file in py/mock.py py/mock-parse-buildlog.py; do
+  sed -i 1"s|#!/usr/bin/python3 |#!%{__python} |" $file
+done
+
+%build
+for i in py/mock.py py/mock-parse-buildlog.py; do
+    perl -p -i -e 's|^__VERSION__\s*=.*|__VERSION__="%{version}"|' $i
+    perl -p -i -e 's|^SYSCONFDIR\s*=.*|SYSCONFDIR="%{_sysconfdir}"|' $i
+    perl -p -i -e 's|^PYTHONDIR\s*=.*|PYTHONDIR="%{python_sitelib}"|' $i
+    perl -p -i -e 's|^PKGPYTHONDIR\s*=.*|PKGPYTHONDIR="%{python_sitelib}/mockbuild"|' $i
+done
+for i in docs/mock.1 docs/mock-parse-buildlog.1; do
+    perl -p -i -e 's|\@VERSION\@|%{version}"|' $i
+done
+
+%install
+#base filesystem
+mkdir -p %{buildroot}%{_sysconfdir}/mock/eol/templates
+mkdir -p %{buildroot}%{_sysconfdir}/mock/templates
+
+install -d %{buildroot}%{_bindir}
+install -d %{buildroot}%{_libexecdir}/mock
+install mockchain %{buildroot}%{_bindir}/mockchain
+install py/mock-parse-buildlog.py %{buildroot}%{_bindir}/mock-parse-buildlog
+install py/mock.py %{buildroot}%{_libexecdir}/mock/mock
+ln -s consolehelper %{buildroot}%{_bindir}/mock
+install create_default_route_in_container.sh %{buildroot}%{_libexecdir}/mock/
+ 
+install -d %{buildroot}%{_sysconfdir}/pam.d
+cp -a etc/pam/* %{buildroot}%{_sysconfdir}/pam.d/
+
+install -d %{buildroot}%{_sysconfdir}/mock
+cp -a etc/mock/* %{buildroot}%{_sysconfdir}/mock/
+
+install -d %{buildroot}%{_sysconfdir}/security/console.apps/
+cp -a etc/consolehelper/mock %{buildroot}%{_sysconfdir}/security/console.apps/%{name}
+
+install -d %{buildroot}%{_datadir}/bash-completion/completions/
+cp -a etc/bash_completion.d/* %{buildroot}%{_datadir}/bash-completion/completions/
+ln -s mock %{buildroot}%{_datadir}/bash-completion/completions/mock-parse-buildlog
+
+install -d %{buildroot}%{_sysconfdir}/pki/mock
+cp -a etc/pki/* %{buildroot}%{_sysconfdir}/pki/mock/
+
+install -d %{buildroot}%{python_sitelib}/
+cp -a py/mockbuild %{buildroot}%{python_sitelib}/
+
+install -d %{buildroot}%{_mandir}/man1
+cp -a docs/mock.1 docs/mock-parse-buildlog.1 %{buildroot}%{_mandir}/man1/
+install -d %{buildroot}%{_datadir}/cheat
+cp -a docs/mock.cheat %{buildroot}%{_datadir}/cheat/mock
+
+install -d %{buildroot}/var/lib/mock
+install -d %{buildroot}/var/cache/mock
+
+mkdir -p %{buildroot}%{_pkgdocdir}
+install -p -m 0644 docs/site-defaults.cfg %{buildroot}%{_pkgdocdir}
+
+sed -i 's/^_MOCK_NVR = None$/_MOCK_NVR = "%name-%version-%release"/' \
+    %{buildroot}%{_libexecdir}/mock/mock
+
+%pre filesystem
+# check for existence of mock group, create it if not found
+getent group mock > /dev/null || groupadd -f -g %mockgid -r mock
+exit 0
+
+%check
+%if %{with lint}
```

Comment:
Can we go ahead and completely remove the linting tests here? As a distribution, this provides us no value.